Library/Formula/fig.rb

```diff
@@ -68,9 +64,10 @@ def install
 
     bin.install Dir[libexec/"bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    ln_s bin/"docker-compose", bin/"fig"
   end
 
   test do
-    system "#{bin}/fig", "--version"
+    system "#{bin}/docker-compose", "help", "build"
```

Comment:
This gives an error exit code, unfortunately: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/22020/version=mountain_lion/testReport/junit/brew-test-bot/mountain_lion/test_fig/